fastcrud/core.py

```diff
@@ -0,0 +1,128 @@
+import typing
+import pydantic
+from fastapi import APIRouter
+from fastcrud.storage import BaseStorage
+from fastcrud.storage.local import LocalStorage
+from fastcrud.utils import run_async_or_sync
+from fastcrud.types import FilterOp
+
+
+class BaseCrud:
+    def __init__(self, model: pydantic.BaseModel, storage: BaseStorage):
+        self.model = model
+        self.storage = storage
+
+        async def get(uid: str):
+            """
+            Get an object by is unique id
```

Comment:
If the user isn't using `sowba.SBaseModel`, they may not have that `uid` field, as such we need to add some error handling along this case. Additionally, we should start to consider error handling conditions.... @oukone do we have a common set of HTTP status codes... I found this reference https://www.codetinkerer.com/2015/12/04/choosing-an-http-status-code.html